tests/phpunit/block_list.php

```diff
@@ -29,7 +29,7 @@ class tool_blocksmanager_block_list_testcase extends advanced_testcase {
     /**
      * Initial set up.
      */
-    public function setUp() {
+    public function setUp() : void {
```

Comment:
This is pretty pedantic of me. There shouldn't be a space between closing bracket and colon. https://docs.moodle.org/dev/Coding_style#Functions_and_Methods